docker/cortx-deploy/cortx-control/Dockerfile

```diff
@@ -51,22 +51,21 @@ RUN yum-config-manager --add-repo ${BUILD_URL}/cortx_iso/ \
     && yum-config-manager --add-repo ${BUILD_URL}/3rd_party \
     && curl ${BUILD_URL}/RELEASE.INFO -o RELEASE.INFO 
 
-COPY ./*.txt ./generate-release-info.sh /opt/seagate/cortx/
+COPY ./cortx-control/*.txt ./common-python-req.txt ./generate-release-info.sh /opt/seagate/cortx/
 
 RUN yum install --nogpgcheck -y python3-pip \
     && pip3 install yq \
-    && pip3 install  --no-cache-dir --trusted-host $(echo $BUILD_URL | awk -F '/' '{print $3}') -i $BUILD_URL/python_deps -r /opt/seagate/cortx/python_requirements.txt -r /opt/seagate/cortx/python_requirements.ext.txt \
+    && pip3 install  --no-cache-dir --trusted-host "$(echo $BUILD_URL | awk -F '/' '{print $3}')" -i $BUILD_URL/python_deps -r /opt/seagate/cortx/python_requirements.txt \
```

Comment:
Do we not need `/opt/seagate/cortx/python_requirements.ext.txt` . Just in case python package has not declared dependency in there package manegemnet.

Reply:
@shailesh-vaidya, Do you mean we should keep `/opt/seagate/cortx/python_requirements.ext.txt` incase component need to declare some deendencies which are not covered in `/opt/seagate/cortx/python_requirements.txt`?

Reply:
Yes. So if dependencies are not installed automatically then install it form `/opt/seagate/cortx/python_requirements.ext.txt`

Reply:
Calling `/opt/seagate/cortx/python_requirements.ext.txt` file from `/opt/seagate/cortx/python_requirements.txt` with `-r` flag.